internal/db/cluster/schema.go

```diff
@@ -0,0 +1,61 @@
+package cluster
+
+import "time"
+
+// CreateSchema is the default schema applied when bootstrapping the database.
+const CreateSchema = `
+CREATE TABLE schemas (
+  id          INTEGER    PRIMARY  KEY    AUTOINCREMENT  NOT  NULL,
+  version     INTEGER    NOT      NULL,
+  updated_at  DATETIME   NOT      NULL,
+  UNIQUE      (version)
+);
+
+CREATE TABLE certificates (
+  id           INTEGER        PRIMARY  KEY    AUTOINCREMENT  NOT  NULL,
+  fingerprint  TEXT           NOT      NULL,
+  type         INTEGER        NOT      NULL,
+  name         TEXT           NOT      NULL,
+  certificate  text           NOT      NULL,
+  UNIQUE       (fingerprint)
+);
+
+CREATE TABLE secrets (
```

Comment:
Maybe call this `tokens` instead of `secrets`?

To me `tokens` assume something ephemeral whereas `secrets` are more permanent.

Reply:
I was hesitant to call it `tokens` because it's not actually the tokens returned from the API that is being stored in the database, rather just a unique record of who we should issue a token for. I suppose I could make it `token_records`?